rest-server/src/templates/yarnContainerScript.mustache

```diff
@@ -69,6 +69,90 @@ docker login --username ${cred[1]} --password ${cred[2]} ${cred[0]} \
   || { echo 'Authorized failed' ; exit 1; }
 {{/ jobData.authFile }}
 
+
+# Get container host list from framework launcher webservice
+function get_host_list()
+{
+  while true; do
+    taskrole_statuses=$(curl -X GET -H "Accept: application/json" {{{ statusUri }}} | jq -r ".aggregatedFrameworkStatus.aggregatedTaskRoleStatuses")
```

Comment:
>statusUri [](start = 70, length = 9)

You can just call:

[http://{addr}/v1/Frameworks/{FrameworkName}/AggregatedFrameworkStatus](http://%7Baddr%7D/v1/Frameworks/%7BFrameworkName%7D/AggregatedFrameworkStatus)

It will reduce the load of launcher and payload.

Reply:
ok